warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -275,6 +279,8 @@
      */
     protected boolean showReducedQueryPrune = true;
     
+    protected FederatedQueryPlanner federatedQueryPlanner = null;
```

Comment:
This feels wrong to me.  Why would the DefaultQueryPlanner need to know about the federated query planner?